holoviews/operation/datashader.py

```diff
@@ -380,6 +380,8 @@ def _process(self, element, key=None):
             category = agg_fn.cat_column
         else:
             category = agg_fn.column if isinstance(agg_fn, ds.count_cat) else None
+        if sel_fn and sel_fn.column is None:
```

Comment:
Should this be limited to `first` and `last` reductions only? I've tried `selector=ds.max()` and it did show something in the tooltip, but I don't understand how it was computed (which column was used to compute the max() row?)

<img width="387" alt="image" src="https://github.com/user-attachments/assets/409d906a-fbb1-48ed-b290-e2b22cd20136" />